calico_kubernetes/calico_kubernetes.py

```diff
@@ -1,23 +1,27 @@
 #!/bin/python
-import json
+from __future__ import print_function
```

Comment:
Importing this tells the compiler to treat `print` as a function and not a keyword. so `print "statement"` becomes `print("statement")`. This makes it easier to unittest.